File: C/1006.c
#include <stdio.h>

int main() {
    double A, B, C, media;

    scanf("%lf", &A);
    scanf("%lf", &B);
    scanf("%lf", &C);


    media = (2*A + B*3 + C*5)/10;

    printf("MEDIA = %.1lf\n", media);

    return 0;
}

File: PHP/1006.php
<?php
    $valor1 = readline();
    $valor2 = readline();
    $valor3 = readline();
    
    $media = ($valor1*2 + $valor2*3 + $valor3*5) /10;
    
    echo "MEDIA = ".number_format($media,1,".","")."\n";
?>